File: scala-gatling/src/gatling/scala/org/openapitools/client/model/MailAttachment.scala

package org.openapitools.client.model

import java.io.File

case class MailAttachment (
    /* Optional filename to specify for the attachment. */
    _filename: Option[String],
    /* Contents of the attached file */
    _data: File
)
object MailAttachment {
    def toStringBody(var_filename: Object, var_data: Object) =
        s"""
        | {
        | "filename":$var_filename,"data":$var_data
        | }
        """.stripMargin
}

File: openapi-client-old/dart-jaguar/lib/model/mail_attachment.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';


part 'mail_attachment.jser.dart';

class MailAttachment {
   /* Contents of the attached file */
  @Alias('data', isNullable: false,  )
  final List<int> data;
   /* Optional filename to specify for the attachment. */
  @Alias('filename', isNullable: false,  )
  final String filename;
  

  MailAttachment(
      

{
    
     this.data = null,   this.filename = null 
    
    }
  );

  @override
  String toString() {
    return 'MailAttachment[data=$data, filename=$filename, ]';
  }
}

@GenSerializer(nullableFields: true)
class MailAttachmentSerializer extends Serializer<MailAttachment> with _$MailAttachmentSerializer {

}

